is_core/forms/fields.py

```diff
@@ -32,11 +32,12 @@ def validate(self, value):
 
 
 class EmptyReadonlyField(ReadonlyField):
+
     widget = EmptyWidget
     readonly_widget = EmptyWidget
 
-    def __init__(self):
-        super(ReadonlyField, self).__init__()
+    def __init__(self, required, label):
+        super(ReadonlyField, self).__init__(required=required, label=label)
```

Comment:
fixed bug. For delete button sometimes label was not translated and field was required